protocols/luci-proto-xfrm/htdocs/luci-static/resources/protocol/xfrm.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+'require uci';
+'require form';
+'require network';
+'require tools.widgets as widgets';
+
+return network.registerProtocol('xfrm', {
+	getI18n: function() {
+		return _('XFRM');
```

Comment:
I think this text is not very meaningful if you do not know exactly what XFRM means.
Maybe` IPsec Tunnel Interface (XFRM)`?
https://github.com/openwrt/openwrt/blob/ec6293febc244d187e71a6e54f44920be679cde4/package/network/config/xfrm/Makefile#L19

Reply:
>  Maybe IPsec Tunnel Interface (XFRM)?

Keep in mind that the description should not be too long to fit into the various UI elements. Maybe `IPsec XFRM` provides enough context. After all, people installing XFRM support and adding XFRM interfaces likely now what it is about.